src/FluentValidation.AspNetCore/FluentValidationMvcExtensions.cs

```diff
@@ -39,15 +39,13 @@ public static class FluentValidationMvcExtensions {
 		///     An <see cref="T:Microsoft.Extensions.DependencyInjection.IMvcCoreBuilder" /> that can be used to further configure the
 		///     MVC services.
 		/// </returns>
-		public static IMvcCoreBuilder AddFluentValidation(this IMvcCoreBuilder mvcBuilder, Action<FluentValidationMvcConfiguration> configurationExpression=null) {
+		public static IMvcCoreBuilder AddFluentValidation(this IMvcCoreBuilder mvcBuilder, Action<FluentValidationMvcConfiguration> configurationExpression = null) {
 			var expr = configurationExpression ?? delegate { };
 			var config = new FluentValidationMvcConfiguration();
 
 			expr(config);
 
-			if (config.AssembliesToRegister.Count > 0) {

```

Comment:
Removed RegisterTypes and replaced it with the call to AddValidatorsFromAssemblies extension method